src/main/java/cz/xtf/radanalytics/driver/build/DriverBuildDefinition.java

```diff
@@ -43,6 +44,9 @@ public DriverBuildDefinition(String appName, String appSubModule, String buildNa
 		// TODO - we can create all builds immediately when starting the testsuite or create specific build on demand(current situation),
 		// probably need to implement a config option for this
 		// createBuilds();
```

Comment:
remove these comments ^^. Put just this one: // we can create all builds immediately when starting the testsuite or create specific build on demand